src/i18n/strings.ftl

```diff
@@ -1542,7 +1542,7 @@ Youre-always-in-control-of-the-location-privacy = You’re always in control of
 # Text prompting the user to open Settings to grant permission after
 # permission has been denied
 Youve-denied-permission-prompt = You’ve denied permission. Please grant permission in the settings app.
-Youve-made-2-observations = You’ve made 2 observations!
+Youve-made-5-observations = You’ve made 5 observations!
```

Comment:
Is there ever a reason we would need to use the `{ $count }` plural rules here? If we did, it would allow us to change our minds on when this pivot card gets triggered without requiring new translations in all languages.